tfbpshiny/misc/bindingmanualqc_table_module.py

```diff
@@ -32,7 +32,7 @@ def bindingmanualqc_table_server(
 
     @render.data_frame
     def bindingmanualqc_table():
-        df_local = bindingmanualqc_df()
+        df_local = bindingmanualqc_df()  # type: ignore
```

Comment:
Type ignore are added to reactive calc or value b/c mypy doesn't do well with shiny objects. If adding any ignore statements to linters or checkers, it must be justified.